packages/dev/parcel-packager-docs/package.json

```diff
@@ -4,9 +4,9 @@
   "private": true,
   "main": "DocsPackager.js",
   "engines": {
-    "parcel": "2.0.0-dev.1599"
+    "parcel": "^2.0.0"
   },
   "dependencies": {
-    "@parcel/plugin": "2.0.0-dev.1601"
+    "@parcel/plugin": "2.13.0"
```

Comment:
same, and more below

i know several of these are internal, but can't hurt to be consistent